src/main/java/zendo/games/zenlib/ZenAssets.java

```diff
@@ -47,6 +47,9 @@ public ZenAssets() {
         preferences = Gdx.app.getPreferences(PREFS_NAME);
 
         mgr = new AssetManager();
+        if (ZenMain.instance.config.uiSkinPath != null) {
```

Comment:
The reason that the config fields aren't static is that I couldn't find a good way to be able to override static values in 'user' code (project code that's using the library, like `zenlib-test`) without making it more confusing. 

Statics are 'owned' by the class rather than an instance, so consider the following:
```java
// library class
public abstract class LibClass {
  public static class Flags {
    public static boolean flag = false;
  }
  public int getIfFlagSet() {
    return Flags.flag ? 1 : 0;
  }  
}

// 'user' class
public class ProjectClass extends LibClass {
  public static class Flags {
    public static boolean flag = true;
  }
}

// in some other 'user' code
public void doHardWorkSometimes(ProjectClass project) {
  if (project.getIfFlagSet() == 1) {
    doHardWork();
  }    
}
```

In this situation, it's not immediately clear which static `flag` determines whether hard work gets done since they both exist at the same time with the same field name and inner class name, but attached to the parent and child classes respectively. It'd be the same deal but maybe even more confusing if it was just a plain static field instead of the static inner class. 

Hopefully this config approach works well for all our uses but I wouldn't be surprised if it needs to be reworked at some point.